packit_service/worker/helpers/testing_farm.py

```diff
@@ -171,12 +171,7 @@ def custom_fmf(self) -> bool:
     def fmf_url(self) -> str:
         return (
             self.job_config.fmf_url
-            or (
-                self.metadata.pr_id
-                and self.project.get_pr(
-                    self.metadata.pr_id
-                ).source_project.get_web_url()
-            )
+            or (self.pr_id and self.pull_request_object.source_project.get_web_url())
```

Comment:
This is a bit strange since `self.pr_id` doesn't seem to be used in the second clause anymore. Can we remove it now?